tests/framework/clients/rancher/v1/client.go

```diff
@@ -52,9 +54,15 @@ type SteveClient struct {
 	steveType string
 }
 
+// NamespacedSteveClient is the client used to access namespaced Steve API endpoints
+type NamespacedSteveClient struct {
+	SteveClient
+	namespace string
+}
+
 type SteveOperations interface {
-	List(opts *types.ListOpts) (*SteveCollection, error)
-	ListAll(opts *types.ListOpts) (*SteveCollection, error)
+	List(params url.Values) (*SteveCollection, error)
+	ListAll(params url.Values) (*SteveCollection, error)
```

Comment:
Nice! I had noticed our Steve client listing functionality was very restrictive and wouldn't work the same as the v3 client when attempting to put a specific values in the  ListOpts.